src/core.js

```diff
@@ -1388,10 +1373,16 @@ define([
 
         if (this._propertiesMug) {
             this._propertiesMug.teardownProperties();
+            try {
+                this._propertiesMug.validate();
+            } catch (err) {
+                // ignore error
```

Comment:
The only case where there is an error here is when a new form is being loaded and the mug no longer resides in the current form, which can cause errors here (completely not important).